lib/src/flare_animation.dart

```diff
@@ -0,0 +1,213 @@
+part of flame_flare;
+
+/// A class that wraps all the settings of a flare animation from [filename].
+///
+/// It has a similar API to the [FlareActor] widget.
+class FlareActorAnimation {
+  FlareActorAnimation(
+    this.filename, {
+    @required double width,
```

Comment:
why not use `this.width` and `this.height`?

Reply:
wilco